aptos-move/aptos-release-builder/src/main.rs

```diff
@@ -34,6 +73,23 @@ pub enum Commands {
         release_config: PathBuf,
         #[clap(short, long)]
         output_dir: PathBuf,
+
+        #[clap(long)]
+        simulate: Option<NetworkSelection>,
+    },
+    /// Simulate a multi-step proposal on the specified network, using its current states.
+    /// The simulation will execute the governance scripts, as if the proposal is already
+    /// approved.
+    SimulateMultiStepProposal {
+        /// Path to the directory storing the scripts (steps) belonging to the proposal.
+        #[clap(short, long)]
+        proposal_dir: PathBuf,
```

Comment:
Can we just pass the output dir herre instead of the specific proposal dir?

So right now when u generate you make the output/ folder with many proposal folders potentially

This means you have to call simulate on each one which might be intentional but is a bit annoying you cannot just pass the same high level path and have this figure it out